MacOSX/build-package.in

```diff
@@ -14,7 +25,7 @@ BUILDPATH=${PWD}
 SDK_PATH=$(xcrun --sdk macosx --show-sdk-path)
 
 # Set SDK path
-export CFLAGS="$CFLAGS -isysroot $SDK_PATH -arch x86_64 -mmacosx-version-min=10.10"
+export CFLAGS="$CFLAGS -isysroot $SDK_PATH -arch x86_64 -mmacosx-version-min=$MACOSX_DEPLOYMENT_TARGET"
```

Comment:
MACOSX_DEPLOYMENT_TARGET
	   If -mmacosx-version-min is unspecified, the default deployment
	   target is read from this environment	variable.

Reply:
done, thanks